src/Handlers/ErrorHandler.php

```diff
@@ -57,8 +57,9 @@ public function handle()
         $exception = $this->logger()->
                             getDataBuilder()->
                             generateErrorWrapper($errno, $errstr, $errfile, $errline);
-
-        $this->logger()->log(Level::ERROR, $exception, array(), true);
+        $exception->isUncaught = true;
```

Comment:
Does it make sense to add `isUncaught` to the `ErrorWrapper` class, or potentially a subclass of `ErrorWrapper` specific to this purpose? You seem to be decorating the exception, if you make it explicit then it'll be very clear what `isUncaught` is meant to do (and you can document it in the decorator you write)